docs/FAQs.md

```diff
@@ -1,5 +1,7 @@
-- [Frequently Asked Questions](#frequently-asked-questions)
-  - [How is Hive different than the OpenShift installer?](#how-is-hive-different-than-the-openshift-installer)
+- [Frequently Asked Questions](#Frequently-Asked-Questions)
```

Comment:
Why are these changes part of this PR?

Reply:
I am just updating the TOC which was broken before. I can move it to a different commit. But it is very small change and only doc changes. Hence thought of keeping it in this PR